packages/ts/tsconfig.json

```diff
@@ -3,7 +3,7 @@
     /* Basic Options */
     "target": "ES2018",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', or 'ESNEXT'. */
     "module": "commonjs",                     /* Specify module code generation: 'commonjs', 'amd', 'system', 'umd' or 'es2015'. */
-    "lib": ["es2018", "ES2020.Promise"],                             /* Specify library files to be included in the compilation:  */
+    "lib": ["es2018", "ES2020.Promise", "ES2021.String"],                             /* Specify library files to be included in the compilation:  */
```

Comment:
`String.ReplaceAll` is used in several pacakges. So, I added it here to avoid surprise.